tests/integration/__init__.py

```diff
@@ -545,9 +563,9 @@ def use_cluster(cluster_name, nodes, ipformat=None, start=True, workloads=None,
 
         # This will enable the Mirroring query handler which will echo our custom payload k,v pairs back
 
-        if 'graph' not in workloads:
-            if PROTOCOL_VERSION >= 4:
-                jvm_args = [" -Dcassandra.custom_query_handler_class=org.apache.cassandra.cql3.CustomPayloadMirroringQueryHandler"]
+        #if 'graph' not in workloads:
```

Comment:
Why this had to be commented out? 
You don't seem to eliminate Cassandra support.

Reply:
jvm_args is what passed down to scylla commandline, and "-Dcassandra.custom_query_handler_class=org.apache.cassandra.cql3.CustomPayloadMirroringQueryHandler" is killing scylla.

you are correct I can handle this more gracefully, i.e. remove it for scylla only

Reply:
Sounds like a plan ;)

Reply:
move to #8